src/machinetalk/config-service/configserver.py

```diff
@@ -267,6 +254,8 @@ def main():
     mki = configparser.ConfigParser()
     mki.read(mkini)
     uuid = mki.get("MACHINEKIT", "MKUUID")
+    aformat = mki.has_option("MACHINEKIT", "ANNOUNCE_FORMAT") and \
+        mki.get("MACHINEKIT", "ANNOUNCE_FORMAT") or None
```

Comment:
Are these being added by your tool since there are spaces? I don’t see the quotes in avahi browser. I’ll look into this one for sure.

Reply:
No, these are part of the domain. When I delete quotes from `machinekit.ini` so the line is just `ANNOUNCE_FORMAT=MK $SRVNAME on $HOSTNAME`, then the domain is all good as expected.

I didn't find any INI file format specifications on the Internet which would be authoritative enough to consider them the law. But what I did find all pointed out that double-quoted strings are voluntarily possible to use and required to use when the user wants white characters on start or end of the string, so I would consider it a bug.

Which is quite easy to resolve in the inifile C++ Machinekit library, but not so much in `six.moves configparser` (did some googling and it's a common issue). I don't think it should be addressed here (because of code cleanliness).

Simplest would be for now to delete quotes from ini file.

Reply:
I've removed the quotes from the ini file to address this since that is consistent with ini file behavior. Good catch!